spec/factories/articles.rb

```diff
@@ -2,5 +2,6 @@
   factory :article do
     title "MyString"
     body "MyText"
+    categories [Category.create(name: 'Fashion')]
```

Comment:
This is not the way you work with associations in FactoryBot. Please take a closer look at https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations

Reply:
This was the main reason the CI build failed! We took it out and it worked! Thanks for the link!

Reply:
We tried several ways of making the association to category, including the association line from the link, but it always says this:
![screen shot 2018-03-18 at 9 22 37 am](https://user-images.githubusercontent.com/34248712/37564020-ec404b90-2a8d-11e8-9789-ee6e746f7462.png)

Using line 5 was the only way we were able to get the tests to pass locally, but it breaks on semaphore which we thought was the problem.

Reply:
I would try something like this to start with:
```ruby
FactoryBot.define do
  factory :article do
    title "MyString"
    body "MyText"
     categories {[FactoryBot.create(:category)]}
  end
end
```